### PR TITLE
New troll DPs and new 1-min addition for DP1.20016.001

### PR DESCRIPTION
**Pull request by ncatolico**
@cklunch

### CODE REVIEW

File: DP1.20008.001/Collection.methodology.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Conductivity of surface water is measured using vented pressure transducers at upstream and downstream sensor sets at wadeable stream sites, on infrastructure near buoys at river sites, and littoral sensor sets at lake sites.  Measurements spatially represent a point and are made at 0.0167 Hz (1 per minute). Data are published as five- and thirty-minute elevation averages.
```
Comment (cklunch) [inline DP1.20008.001/Collection.methodology.txt:1]: Elevation averages? I think the text here might be meant for surface water elevation

**Review by cklunch — COMMENTED** [review]
@ncatolico Looking good! See comments for specific edit, all the changes should be easy. I assume we should wait to publish these until the new products come out from Pachyderm?

**Comment by ncatolico** [comment]
@cklunch, thanks for the detailed review! Updated files submitted. It would be good to publish them prior to the PROD pachyderm release since we are in the CERT testing phase currently. I'd like to make sure the download package looks as expected. These data products in PROD are still listed as "FUTURE" availability which would still prevent users from downloading.

**Comment by cklunch** [comment]
@ncatolico Got it. Thanks for the edits, looks great. I'll merge these changes and publish to the CERT portal. They'll sync tonight, and then you may need to request a cache refresh tomorrow, I don't think they refresh the CERT portal every day.

**Pull request merged**